env.example

```diff
@@ -2,7 +2,10 @@
 // This is only a .env example
 // Do not change this file, adding your GITHUB TOKEN here!
 // Use cp or mv (like this: cp env.example .env), then edit .env with your GITHUB TOKEN.
-// IMPORTANT: Don't forget to add to update your .gitignore with .env (to avoid public share your key!)
+// IMPORTANT: Don't forget to add to update your .gitignore with .env (to avoid making your key public!)
 
 REACT_APP_GITHUB_TOKEN = "YOUR GITHUB TOKEN HERE"
 GITHUB_USERNAME = "YOUR GITHUB USERNAME HERE"
+// Set to true to fetch profile data from GitHub (remember to remove all components relying on GitHub data if set to false)
+// Ensure data type is string
+USE_GITHUB_DATA = "false"
```

Comment:
add a new line here.
Also, I think, since it's an example file I would set the default value to be `true`, what do you think?

Reply:
I set it to false as default so that new users can setup dev environment without crashes and don't feel discouraged that it isn't working. 
However, keeping it to true is also an equally good decision as then it would throw errors on first run which would enforce people to setup these variables.

Reply:
> I set it to false as default so that new users can setup dev environment without crashes and don't feel discouraged that it isn't working.
On the first run, I think this file wouldn't be read at all. The file will only be read when it is changed to `.env`, which means the user has already made some efforts to make things work and it would be good to set it as true, and that won't be discouraging.

Reply:
Yep, set it to true but now we're assuming they've followed the new added steps to copy the env.example file before running the app locally so keeping that in mind too, setting it to default as true is good.